docs/network protocol.md

```diff
@@ -1,19 +1,7 @@
 # Archipelago General Client
-## Archipelago Connection Handshake
-These steps should be followed in order to establish a gameplay connection with an Archipelago session.
-
-1. Client establishes WebSocket connection to Archipelago server.
-2. Server accepts connection and responds with a [RoomInfo](#RoomInfo) packet.
-3. Client may send a [GetDataPackage](#GetDataPackage) packet.
-4. Server sends a [DataPackage](#DataPackage) packet in return. (If the client sent GetDataPackage.)
-5. Client sends [Connect](#Connect) packet in order to authenticate with the server.
-6. Server validates the client's packet and responds with [Connected](#Connected) or [ConnectionRefused](#ConnectionRefused).
-7. Server may send [ReceivedItems](#ReceivedItems) to the client, in the case that the client is missing items that are queued up for it.
-8. Server sends [PrintJSON](#PrintJSON) to all players to notify them of the new client connection.
-
-In the case that the client does not authenticate properly and receives a [ConnectionRefused](#ConnectionRefused) then the server will maintain the connection and allow for follow-up [Connect](#Connect) packet.
-
-There are also a number of community-supported libraries available that implement this network protocol to make integrating with Archipelago easier.
+There are a number of community-supported libraries available that implement the Archipelago network protocol to make integrating with Archipelago easier.
+This document is intended for people who are creating a new library, 
+contributing code to an existing library, or handling all the networking directly in a client (not recommended).
```

Comment:
This is wrong. People using a lib may need to understand how the communication works.